executor/envd/api/state.py

```
@@ -54,20 +61,30 @@ def init(self, hyperloop_ip: Optional[str], env_vars: Optional[Dict[str, str]],
             if not should_update:
-                logger.info(f"Skipping update: request timestamp {request_time} is not newer than last set time {self.last_set_time}")
+                logger.info(
+                    f"Skipping update: request timestamp {request_time} is not newer than last set time {self.last_set_time}"
+                )
                 return
 
             # Check for access token conflict
             if access_token and self.access_token and self.access_token != access_token:
-                logger.warning(f"Access token conflict: attempting to set new token when one already exists")
+                logger.warning(
+                    f"Access token conflict: attempting to set new token when one already exists"
```

Ruff (0.14.13)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)